addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -135,11 +137,7 @@
                                                         <td width="30%" align="right">
                                                             <div>
                                                                 <span t-field="ml.location_dest_id"/>
-                                                            </div>
-                                                            <div t-if="ml.result_package_id or ml.result_package_id">
-                                                                <span t-field="ml.package_id"/>
-                                                                <span t-if="ml.result_package_id and not ml.package_id" class="fa fa-dropbox"/>
-                                                                <span t-if="ml.result_package_id and ml.package_id">&#8594;</span>
+                                                                <span t-if="ml.result_package_id"> - </span>
```

Comment:
[REF] stock: packages and report

- delivery slip: remove mention of packages
- package: fix the report's  traceback
- picking: moved the packages references to a table at the bottom of the report